sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducer/EventHubProducerLiveTests.cs

```diff
@@ -855,5 +857,32 @@ await using (var producer = client.CreateProducer())
                 }
             }
         }
+
+#if FullNetFx
```

Comment:
Same question as the consumer... do we still need this ugly thinger?

Reply:
In fact, it does work for Core. I have removed the preprocessor directives.